pootle/apps/pootle_project/models.py

```diff
@@ -388,7 +388,21 @@ def resources(self):
     def __unicode__(self):
         return self.fullname
 
+    def clean(self):
+        self.code = self.code.strip()
+
+        if not self.code:
+            raise ValidationError("Blank strings cannot be used as a project "
+                                  "code")
+
+        if self.code in RESERVED_PROJECT_CODES:
```

Comment:
This is also present in the project form. Given that such form will use this model, doesn't this validation already kick in? In other words, isn't the duplicated validation in `ProjectForm` unnecessary?